mitosheet/src/mito/components/toolbar/Toolbar.tsx

```diff
@@ -168,7 +174,7 @@ export const Toolbar = (
                     />
                 </div>
             </div>
-            {currentTab !== undefined ? tabs[currentTab] ?? <div> No tab found </div> : undefined}
+            {currentTab !== undefined ? tabs[currentTab] ?? tabs['Home'] : undefined}
```

Comment:
Ah -- so if you close a graph, this ends up being undefined. So we just default to the Home. 

This is reasonable -- but wanna leave a comment about why this happens (so that we can just not display a tab like something for charting and it will not break everything).